apps/src/bounce/visualization.html.ejs

```diff
@@ -1,5 +0,0 @@
-<svg xmlns="http://www.w3.org/2000/svg" version="1.1" id="svgBounce">
-</svg>
-<div id="capacityBubble">
-  <div id="capacity"></div>
-</div>
```

Comment:
Note, afaict capacity/capacityBubble aren't used anywhere, so I got rid of these from a couple of apps.